tests/client/test_v1.py

```diff
@@ -0,0 +1,24 @@
+"""Tests for Shaarli REST API v1 client"""
+from unittest import mock
+
+from shaarli_client.client import ShaarliV1Client
+
+SHAARLI_URL = 'http://domain.tld/shaarli/'
+SHAARLI_SECRET = 's3kr37!'
+
+
+def test_constructor():
+    """Instantiate a new client"""
+    ShaarliV1Client(SHAARLI_URL, SHAARLI_SECRET)
+
+
+@mock.patch('requests.request')
```

Comment:
Thanks for the clarification, I will definitely re-read my docs on python decorators